src/lib/funding.js

```diff
@@ -113,3 +116,28 @@ export function precacheRememberedFunding() : ZalgoPromise<void> {
     }
     return ZalgoPromise.resolve();
 }
+
+export function getRenderedButtons(props : Object) : string {
```

Comment:
Please avoid using `Object` type, it provides no type safety :)

Reply:
I was basing it on what I had seen elsewhere. https://github.com/paypal/paypal-checkout-components/blob/v4/src/button/validate.js%20#L54

Should there be a defined propsType instead?

Reply:
@ssono could we leverage the `normalizeProps` function here? https://github.com/paypal/paypal-checkout-components/blob/v4/src/button/props.js#L60. It seems like it returns `sources` and eliminates the need for alot of this logic in `getRenderedButtons`

Reply:
Yes we can do this @gregjopa